CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 ## v73.0.0-SNAPSHOT - unreleased
 
 * Support for reporting Client Version in Admin WebSockets tab.
+* ⚠️ Removed TS rule from DashViewSpec that allowed any type of additional values to be added to a DashViewSpec.
+  The rule defeats the goal of the DashViewSpec interface, which is to help guide app developers
```

Comment:
TMI, I think.  Ok to just say "Removed Incorrect TypeScript constraint..." and let the chips fall where they may